package.json

```diff
@@ -17,11 +17,11 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev",
-    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev-cloud",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
-    "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
+    "start:dev:oss": "yarn start:dev --hot --live-reload --static-watch --target 'web'",
```

Comment:
TODO: these extra flags are because the live reload is not working. (Have to manual refresh the browser.)

This change is contained within the [latest commit](https://github.com/influxdata/ui/pull/6621/commits/c6b204ca8f4af03471f9f10aa6536088b724323c), such that we can undo after debugging.